darwin/future/data_objects/item.py

```diff
@@ -75,7 +75,7 @@ def validate_fps(cls, values: dict) -> dict:
         elif isinstance(value, (int, float)):
             type = values.get("type")
             if type == "image":
-                assert value == 0, "fps must be 0 for images"
+                assert value == 0 or value == 1.0, "fps must be '0' or '1.0' for images"
```

Comment:
turns out on our api, 1.0 also exists for items that are images. Found in dogfooding.